Adding  option .directory for scaffolding templates (#2985)
tuist/tuist: projects/docs/docs/commands/scaffold.md

If `.string` and `.files` don't provide enough flexibility, you can leverage the [Stencil](https://github.com/stencilproject/Stencil) templating language via the `.file` case. Besides that, you can also use additional filters defined [here](https://github.com/SwiftGen/StencilSwiftKit#filters)

> Additionally is given `.directory` which gives the possibility to copy entire folders from one path inside another.

Reviewer: ```suggestion
You can also use `.directory` which gives the possibility to copy entire folders to a given path.
```

Author: Changed, thanks :)
